### ISSUE TITLE
Returning from installation does not continue asdf

### ISSUE THREAD

**Issue by ejpcmac** [asdf-vm/asdf-erlang#57]
After a successful installation, `asdf` does not continue with the other plugins. It was the case before `kerl` integration.

## Steps to reproduce

1. Create a `.tool-versions` containing an Erlang and an Elixir version, both non installed
2. Run `asdf install`

## Actual result

`asdf` calls `kerl` to build and install Erlang, and then exits. Elixir version is not installed.

## Expected result

`asdf` would continue its installation, running other plugins.

**Comment by mconigliaro** [comment]
I just ran into the same problem. Moving `erlang <version>` to the bottom of the file seems to work for now.

**Comment by ejpcmac** [comment]
Maybe it is related to the `set -e` in the install script. But that means there is a problem somewhere else. I will try to add logs an find from where it comes tomorrow if I have some time.

**Comment by ejpcmac** [comment]
Ok, so I’ve just uninstalled and installed again Erlang 20.2 on my machine several times, phew! I come up with an insteresting observation.

## What I’ve done

1. I uninstalled Erlang 20.2 and set a `.tool-version` with `erlang 20.2` prior to `elixir 1.6.0-otp-20`.
2. To avoid re-building on each reinstall, I commented out the `kerl cleanup` in the install script.
3. I ran `asdf install`.
4. After building and installing Erlang, `asdf` returned without checking for the Elixir installation.
5. In the uninstall script, I commented out the `kerl delete build`.
6. I ran `asdf uninstall erlang 20.2`.
7. I commented out `kerl build` in the install script and ran `asdf install` again.
8. It installed Erlang 20.2 and checked for the Elixir installation (so it works in this case).
9. I uninstalled Erlang 20.2 again with the modified uninstall script.
10. I uncommented the `kerl cleanup` in the install script and launch the installation again.
11. After installing Erlang 20.2, `asdf` checked for the Elixir installation (so that’s not the cleanup).
12. I unpatched both scripts, uninstalled and re-installed Erlang 20.2 to be sure: issue again.

## My conclusion

It must come from the `kerl build` script doing some side effect.

**Comment by lleger** [comment]
I experienced this as well with Erlang 20.3 and Elixir 1.6.4. The culprit in my case, I think, is the absence of the Java compiler. Even with appropriate Kerl flags, it still fails to build Elixir the first time.

**Comment by Stratus3D** [comment]
Is this still an issue? I just ran `asdf install` with a `.tool-versions` file that contained erlang followed by elixir, and both installed successful. I'm going to close this issue. Please comment if you are still having issues with the latest version of asdf and asdf-erlang.

**Comment by llenrique** [comment]
```shell
cat .tool-versions 
erlang 19.3.6
elixir 1.7.2-otp-19
➜   git:(enrique/adds-publisher) ✗ asdf install      
elixir 1.7.2-otp-19 is already installed
asdf_19.3.6 is not a kerl-managed Erlang/OTP installation
The asdf_19.3.6 build has been deleted
Extracting source code
Building Erlang/OTP 19.3.6 (asdf_19.3.6), please wait...
Configure failed.
config.status: creating make/output.mk
config.status: creating make/emd2exml
=== configuring in lib (/home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6/lib)
configure: running /bin/bash /home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6/lib/configure --disable-option-checking '--prefix=/usr/local'  '--disable-debug' '--without-javac' 'ERL_TOP=/home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6' --cache-file=/dev/null --srcdir=/home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6/lib
configure: creating ./config.status
=== configuring in snmp/. (/home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6/lib/snmp/.)
configure: running /bin/bash ./configure --disable-option-checking '--prefix=/usr/local'  '--disable-debug' '--without-javac' 'ERL_TOP=/home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6' --cache-file=/dev/null --srcdir=.
configure: error: cannot find required auxiliary files: config.guess config.sub
configure: error: ./configure failed for snmp/.
configure: error: /home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_src_19.3.6/lib/configure failed for lib

Please see /home/albacanis/.asdf/plugins/erlang/kerl-home/builds/asdf_19.3.6/otp_build_19.3.6.log for full details.
```

**Comment by Stratus3D** [comment]
@llenrique did you look at that log file for full details? Are you certain you installed all the packages required by Erlang 19 for your OS?

**Comment by llenrique** [comment]
@Stratus3D I solved it by installing erlang 25.2.1.
